index.js

```diff
@@ -149,6 +149,10 @@ module.exports = async (input = 'patch', options) => {
 
 	tasks.add({
 		title: 'Pushing tags',
+		skip: async () => {
+			const {stdout} = await execa('git', ['status', '-sb']);
+			return /^##\s\w+$/.test(stdout.split('\n')[0]);
```

Comment:
Would be good to have a comment here about its purpose as it's not clear from just reading the code.